diagnostic.md

```diff
@@ -5,74 +5,79 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The backend allows multiple users to interact with a site and its data at once. It also stores data so that users can save/retrieve
+things between sessions
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Because views are handled in the client
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create, Read, Update, Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+The Model
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+GET, PUT, PATCH, POST, DELETE
```

Comment:
These are the HTTP verbs, we were looking for Index, Show, Update, Create, and Destroy.